### ISSUE TITLE
Conflict with Meta Box AIO

### ISSUE THREAD

**Issue by zimmiel** [soflyy/oxygen-bugs-and-features#2805]
After I updated Oxygen 4.0 Beta 2 got conflict with Meta Box AIO
I tested on Local Host (PHP 8, MySQL 8.0.16)
![oxygen-beta2](https://user-images.githubusercontent.com/42843497/161384373-e0852efd-0c56-4605-911c-903d6e05af0b.jpg)

**Comment by Schokolader** [comment]
Yes, Metabox Dynamic Data is not available anymore. https://share.getcloudapp.com/YEurO8G6

**Comment by localsbm** [comment]
I get the same error when on PHP8, no error on PHP 7.4 though.
